packages/fluentui/accessibility/src/behaviors/Tree/treeItemAsListItemBehavior.ts

```diff
@@ -5,16 +5,21 @@ import { treeTitleAsListItemTitleBehavior } from './treeTitleAsListItemTitleBeha
 
 /**
  * @description
- * Adds role 'listitem' to a non-leaf item and 'none' to a leaf item.
+ * For a non-leaf item:
+ * - Adds role 'option' when it is selectable
+ * - Adds role 'listitem' when it is none-selectable
+ * Adds role 'none' to a leaf item.
  */
 export const treeItemAsListItemBehavior: Accessibility<TreeItemBehaviorProps> = props => {
   const behavior = treeItemBehavior(props);
+  delete behavior.attributes.root['aria-checked'];
 
   const definition = _.merge(behavior, {
     attributes: {
       root: {
         ...(props.hasSubtree && {
-          role: 'listitem',
+          role: props.selectable ? 'option' : 'listitem',
+          ...(props.selectable && { 'aria-selected': !!props.selected }),
```

Comment:
aria-checked?

Reply:
YeaH :D believe me I did not forget. Somehow `aria-checked` refuse to work with Mac VoiceOver. I have a screen shot in the PR description for it. Since `treeAsListBehavior` is mostly for Mac, so I went with `aria-selected`, which narrates properly with VO

Reply:
Should we keep only `aria-selected` then @kolaps33 for consistence?

Reply:
Notes upon resolving this discussion:
PR #15142 changes all the `aria-selected` to `aria-checked` in order to provide a better experience for JAW users.
This PR keeps `aria-selected` because `aria-checked` does not work properly with VoiceOver.
Consistency is sacrificed for better screen reader users' experience